common/djangoapps/student/urls.py

```diff
@@ -0,0 +1,55 @@
+"""
+URLs for student app
+"""
+
+from django.conf import settings
+from django.conf.urls import patterns, url
+
+from student.views import LogoutView
+
+urlpatterns = (
+    'student.views',
+
+    url(r'^logout$', LogoutView.as_view(), name='logout'),
+    # TODO: standardize login
+    # login used by cms. ajax view that actually does the work
```

Comment:
I just move the existing comment.  I don't know.  I could look into this further I guess.  I was just trying to merge. :)